examples/api200/logical_switch.rb

```diff
@@ -12,22 +12,23 @@
 require_relative '../_client'
 
 # SSH Credential
-ssh_credentials = OneviewSDK::LogicalSwitch::CredentialsSSH.new('dcs', 'dcs')
+ssh_credentials = OneviewSDK::LogicalSwitch::CredentialsSSH.new(@logical_switch_ssh_user, @logical_switch_ssh_password)
 
 # SNMP credentials
-snmp_v1 = OneviewSDK::LogicalSwitch::CredentialsSNMPV1.new(161, 'public')
-snmp_v1_2 = OneviewSDK::LogicalSwitch::CredentialsSNMPV1.new(161, 'public')
+snmp_v1 = OneviewSDK::LogicalSwitch::CredentialsSNMPV1.new(161, @logical_switch_community_string)
+snmp_v1_2 = OneviewSDK::LogicalSwitch::CredentialsSNMPV1.new(161, @logical_switch_community_string)
 
+logical_switch_group = OneviewSDK::LogicalSwitchGroup.get_all(@client).first
 
 logical_switch = OneviewSDK::LogicalSwitch.new(
   @client,
   name: 'Teste_SDK',
```

Comment:
I know it was not added right now, but could we change the name of this LogicalSwitch?